docs/mkdocs/docs/tutorial/metrics.md

```diff
@@ -39,6 +43,86 @@ The descriptive statistics are calculated per feature on any given dataset.
 
     If actuals aren't provided for all inferences, then **ONLY the inferences that have actuals** will be used for the calculation of the drifting metrics.
 
+### Data drift
+
+An analysis happens comparing the current data to the reference data estimating the distributions of each feature in the two datasets. The schema of both datasets should be identical.
+
+Returns a drift summary of the following form:
+
+```
+{'timestamp': the timestamp of the report,
+ 'drift_summary':
+    {'number_of_columns': total number of dataset columns,
+     'number_of_drifted_columns': total number of drifted columns,
+     'share_of_drifted_columns': ('number_of_drifted_columns/'number_of_columns'),
+     'dataset_drift': Boolean based on the criteria below,
+     'drift_by_columns':
+        {'column1': {'column_name': 'column1',
+                     'column_type': the type of column (e.g. num),
+                     'stattest_name': the statistical test tha was used,
+                     'drift_score': the drifting score based on the test,
+                     'drift_detected': Boolean based on the criteria below,
+                     'threshold': a float number based on the criteria below},
+                    {......}
+        }
+    }
+}
+```
```

Comment:
This should be json. Have a look at what I did to the others.

Reply:
Done